src/server/actions/reactivateUser.js

```diff
@@ -0,0 +1,31 @@
+import config from 'src/config'
```

Comment:
This server action should have a test added. I'd recommend using stubs & spies for the internal services and just checking to see that certain functions were called. See test examples for the `deactivateUser` action.